subprojects/docs/src/docs/userguide/authoring-builds/more_about_tasks.adoc

```diff
@@ -1368,8 +1368,6 @@ include::{snippetsPath}/tasks/finalizersWithFailure/tests-groovy/taskFinalizersW
 ----
 ====
 
-On the other hand, finalizer tasks are not executed if the finalized task didn't do any work, for example if it is considered up to date or if a dependent task fails.
```

Comment:
Finalizer tasks are not executed if the finalized tasks did not run - for example when a dependent task of the finalized task failed. I wonder if we need to spell this out here or not. I suppose deleting that sentence is fine.

Reply:
Ok, I will merge this then.